cmd/ops/arg_validation_helpers.go

```diff
@@ -0,0 +1,25 @@
+package ops
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func exactlyNArgsValidator(n int, errMsg string) cobra.PositionalArgs {
```

Comment:
wdyt of having imperative names? `validateExactArgsCount` and `validateMinimumArgsCount`

Reply:
With Go being a imperative language I would think that works well :D

Reply:
Good point about the name, I neither made it Goish not Cobraish. Looking and the [built-in validators](https://github.com/spf13/cobra/blob/main/user_guide.md#positional-and-custom-arguments) the names should be:

- ExactWithErrorArgs(int, string) or simply ExactArgs
- MinimumNWithErrorArgs(int, string) or simply MinimumNArgs

What do you think is the best? What is clear is that I don't like the name I gave them now ;)

Reply:
I'd say `ExactArgs`, the fact that it's a validator already kinda implies there might be errors